drupal8-apache/7.0/usr/local/share/container/baseimage-30.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+alias_function do_build do_drupal8_build_inner
```

Comment:
trouble is do_php_apache_build sounds like it should be doing stuff related to that layer, but it's doing the upstream base image's work.

As `container php_apache_build` is possible to be run, that could be confusing.

Alternatively it might be possible to reduce the function aliasing to something simpler. I'd have to see what's possible